File: xFit/xFit.Model/Korisnik.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace xFit.Model
{
	public class Korisnik
	{
		public int KorisnikId { get; set; }

		public string? Ime { get; set; }

		public string? Prezime { get; set; }

		public DateTime? DatumRodjenja { get; set; }
		public byte[]? Slika { get; set; }
		public string? Email { get; set; }
		public string? Telefon { get; set; }
		public string? Adresa { get; set; }


		public string? KorisnickoIme { get; set; }

		
		public virtual ICollection<KorisnikUloga> KorisnikUlogas { get; } = new List<KorisnikUloga>();

		
	}
}

File: xFit/UI/xfit_mobilna/lib/models/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  int? korisnikId;
  String? ime;
  String? prezime;
  DateTime? datumRodjenja;
  String? slika;
  String? email;
  String? telefon;
  String? adresa;
  String? korisnickoIme;


  Korisnik(this.korisnikId, this.ime, this.prezime, this.datumRodjenja,this.slika,this.email,this.telefon,this.adresa,this.korisnickoIme);

  factory Korisnik.fromJson(Map<String, dynamic> json) => _$KorisnikFromJson(json);

  Map<String, dynamic> toJson() => _$KorisnikToJson(this);
}